examples/simple/simple.mk

```diff
@@ -0,0 +1,91 @@
+QEMU := qemu-system-aarch64
+
+MICROKIT_TOOL ?= $(MICROKIT_SDK)/bin/microkit
+
+BOARD_DIR := $(MICROKIT_SDK)/board/$(MICROKIT_BOARD)/$(MICROKIT_CONFIG)
+SYSTEM_DIR := ${EXAMPLE_DIR}/board/$(MICROKIT_BOARD)
+SYSTEM_FILE := ${SYSTEM_DIR}/simple.system
+IMAGE_FILE := loader.img
+REPORT_FILE := report.txt
+
+vpath %.c ${VMM} ${EXAMPLE_DIR}
+
+IMAGES := vmm.elf
+
+CFLAGS := \
+	  -mstrict-align \
+	  -ffreestanding \
+	  -g3 -O3 -Wall \
+	  -Wno-unused-function \
+	  -DMICROKIT_CONFIG_$(MICROKIT_CONFIG) \
+	  -DBOARD_$(MICROKIT_BOARD) \
+	  -DCONFIG_$(MICROKIT_CONFIG) \
+	  -I$(BOARD_DIR)/include \
+	  -I${VMM}/src \
+	  -I${VMM}/src/arch/aarch64 \
+	  -I${VMM}/src/util \
+	  -I$(SDDF)/include \
+	  -MD \
+	  -MP \
+	  -target $(TARGET)
+
+LDFLAGS := -L$(BOARD_DIR)/lib
+LIBS := --start-group -lmicrokit -Tmicrokit.ld libvmm.a --end-group
+
+CHECK_FLAGS_BOARD_MD5:=.board_cflags-$(shell echo -- ${CFLAGS} ${BOARD} ${MICROKIT_CONFIG} | shasum | sed 's/ *-//')
+
+${CHECK_FLAGS_BOARD_MD5}:
+	-rm -f .board_cflags-*
+	touch $@
+
+vmm.elf: vmm.o images.o
+	$(LD) $(LDFLAGS) $^ $(LIBS) -o $@
+
+all: loader.img
+
+-include vmm.d
+
+${IMAGES}: libvmm.a
+
+${IMAGE_FILE} $(REPORT_FILE): $(IMAGES) $(SYSTEM_FILE)
+	$(MICROKIT_TOOL) $(SYSTEM_FILE) --search-path $(BUILD_DIR) --board $(MICROKIT_BOARD) --config $(MICROKIT_CONFIG) -o $(IMAGE_FILE) -r $(REPORT_FILE)
+
+rootfs.cpio.gz: $(SYSTEM_DIR)/rootfs.cpio.gz
+	$(VMM)/tools/packrootfs $(SYSTEM_DIR)/rootfs.cpio.gz rootfs -o $@
+
+vm.dts: $(SYSTEM_DIR)/linux.dts $(SYSTEM_DIR)/overlay.dts
+	$(VMM)/tools/dtscat $^ > $@
+
+vm.dtb: vm.dts
+	$(DTC) -q -I dts -O dtb $< > $@
+
+vmm.o: ${EXAMPLE_DIR}/vmm.c ${CHECK_FLAGS_BOARD_MD5}
+	$(CC) $(CFLAGS) -c -o $@ $<
+
+images.o: $(VMM)/tools/package_guest_images.S $(SYSTEM_DIR)/linux vm.dtb rootfs.cpio.gz
+	$(CC) -c -g3 -x assembler-with-cpp \
+					-DGUEST_KERNEL_IMAGE_PATH=\"$(SYSTEM_DIR)/linux\" \
+					-DGUEST_DTB_IMAGE_PATH=\"vm.dtb\" \
+					-DGUEST_INITRD_IMAGE_PATH=\"rootfs.cpio.gz\" \
+					-target $(TARGET) \
+					$(VMM)/tools/package_guest_images.S -o $@
+
+include ${VMM}/vmm.mk
+
+qemu: $(IMAGE_FILE)
+	# @ivanv: check that the amount of RAM given to QEMU is at least the number of RAM that QEMU is setup with for seL4.
```

Comment:
Can you remove this line (even though you didn't add it)?

Reply:
Done